.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -1,8 +1,9 @@
 ---
 name: Enhancement Request
 description: Is something critical missing? Suggest an enhancement.
-labels:
-  - enhancement
+title: 'Please add a short decsription.'
+labels: ["enhancement", "needs-review"]
+projects: ["vmware/22"]
 body:
   - type: markdown
     attributes:
```

Comment:
Do we need the empty lines - 13 and 14?

Reply:
Yes, otherwise the form text gets "squished" when it renders.